e2e/linodemachine-controller/minimal/02-create-linodemachine.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      MACHINE_UID="$(OBJ=machines/machine-sample make getKubeUid)" \
+      TC="$(TPL="$PWD/02-create-linodemachine.tpl.yml" make renderTestCase)" make runTestCase
```

Comment:
While running the `e2etest` target locally, kuttl failed at this step:
```
=== CONT  kuttl/harness/minimal#02
    logger.go:42: 13:00:28 | minimal/1-create-cluster | LinodeCluster:kuttl-test-topical-moray/linodecluster-sample created
    logger.go:42: 13:00:28 | minimal/1-create-cluster | Cluster:kuttl-test-topical-moray/cluster-sample created
    logger.go:42: 13:00:28 | minimal/1-create-cluster | Machine:kuttl-test-topical-moray/machine-sample created
    logger.go:42: 13:00:28 | minimal/1-create-cluster | Secret:kuttl-test-topical-moray/bootstrap-data-sample created
    logger.go:42: 13:00:28 | minimal/1-create-cluster | test step completed 1-create-cluster
    logger.go:42: 13:00:28 | minimal/2-create-linodemachine | starting test step 2-create-linodemachine
    logger.go:42: 13:00:28 | minimal/2-create-linodemachine | running command: [sh -c MACHINE_UID="$(OBJ=machines/machine-sample make getKubeUid)" \
        TC="$(TPL="$PWD/02-create-linodemachine.tpl.yml" make renderTestCase)" make runTestCase]
    logger.go:42: 13:00:28 | minimal/2-create-linodemachine | Error: no test directories provided, please provide either --config or test directories on the command line
    logger.go:42: 13:00:28 | minimal/2-create-linodemachine | make[2]: *** [runTestCase] Error 1
    case.go:364: failed in step 2-create-linodemachine
    case.go:366: exit status 2
    logger.go:42: 13:00:28 | minimal | minimal events from ns kuttl-test-topical-moray:
    logger.go:42: 13:00:28 | minimal | Deleting namespace: kuttl-test-topical-moray
```

Reply:
@bcm820 

You can test rendering part with this commands, maybe it helps find the problem:
```bash
e2e/linodemachine-controller/minimal
❯ TPL=02-create-linodemachine.tpl.yml make renderTestCase 
/run/user/51170/tmp.1GKQUKF99C
e2e/linodemachine-controller/minimal
❯ cat /run/user/51170/tmp.1GKQUKF99C/step/00-step.yaml 
apiVersion: infrastructure.cluster.x-k8s.io/v1alpha1
kind: LinodeMachine
metadata:
  ownerReferences:
  - apiVersion: cluster.x-k8s.io/v1beta1
    kind: Machine
    name: machine-sample
    uid: 
  name: linodemachine-sample
spec:
  region: us-sea
  type: g5-nanode-1
```